src/rulesets/Base/mapreduce.jl

```diff
@@ -57,13 +57,6 @@ function frule(::typeof(sum), x)
     return sum(x), sum_pushforward
 end
 
-function rrule(::typeof(sum), x::AbstractArray{<:Real})
```

Comment:
Is there a reason why this function is being removed? You didn't mention in the CR body.

Reply:
Nvm, I can see it in your commit message.